docs/docs/exosphere/state-manager-setup.md

```
@@ -0,0 +1,374 @@
+   # Check container status
+   docker-compose ps
+   
+   # Check logs
+   docker-compose logs -f api-server
+   
+   # Test the API
+   curl http://localhost:8000/health
+   ```
+
+The state manager will be available at:
+- **API**: http://localhost:8000
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)